packages/server/database/types/TimelineEventTeamPromptComplete.ts

```diff
@@ -0,0 +1,27 @@
+import TimelineEvent from './TimelineEvent'
+
+interface Input {
+  id?: string
+  createdAt?: Date
+  interactionCount?: number
+  seenCount?: number
```

Comment:
-1 These fields appear to be never written to. I think we should store the count here if it's static and also make those fields mandatory and not optional.

Reply:
These default to zero in the `TimelineEvent` model, so I'm just removing them entirely here.